packages/pkl.tar/examples/test_examples.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash -e
+
+# test the files archive.*.pkl in this directory
+# evaluate each and ask tar to list the result's contents
+
+cd "$(dirname "$0")"
+
+for file in archive.*.pkl; do
+  echo "+ pkl eval $file | tar tv"
+  pkl eval $file | tar tv -v
+  echo
+done
+
+echo "PASS"
```

Comment:
Good test! Let's add a one-off Gradle task to run this:

```kt
val testTarExamples by tasks.registering(Exec::class) {
  executable = file("packages/pkl.tar/examples/test_examples.sh").absolutePath

  inputs.files(fileTree("packages/pkl.tar/"))

  val outputFile = layout.buildDirectory.file("testTarExample.txt")
  outputs.file(outputFile)

  doLast {
    outputFile.get().asFile.writeText("OK")
  }
}

tasks.test {
  dependsOn(testTarExamples)
}
```

Reply:
What's the best way to handle no `pkl` executable being present in CI? Should we just install one? Or is there a way to set an env var to ask the script to exec `jpkl` out of the class path somehow?